lib/X25519KeyPair.js

```diff
@@ -170,19 +170,57 @@ export class X25519KeyPair extends LDKeyPair {
 
   /**
    * Adds a public key base to a public key node.
+   * Used by LDKeyPair.publicNode(), to serialize the public key material.
    * @example
-   * > keyPair.addEncodedPublicKey({});
-   * { publicKeyBase58: 'GycSSui454dpYRKiFdsQ5uaE8Gy3ac6dSMPcAoQsk8yq' }
-   * @param {Object} publicKeyNode - The public key node
+   * > keyPair.publicNode();
+   * {
+   *   id: 'did:example:1234#z6LSbh9HiAU2zzBdFMdKZGHfg1UjvAYF8C8kYnkfGKuCxYEB',
+   *   type: 'X25519KeyAgreementKey2019',
+   *   controller: 'did:example:1234',
+   *   publicKeyBase58: '1y8BrfAuXTt9yFZ2cmiMRGG5218Raxbfp2ymsFgFATR'
+   * }
+   * @param {object} publicKeyNode - The public key node
    * @param {string} publicKeyNode.publicKeyBase58 - Base58 Public Key
    *
-   * @returns {{verify: Function}} A PublicKeyNode in a block.
+   * @returns {object} A PublicKeyNode in a block.
    */
   addEncodedPublicKey(publicKeyNode) {
     publicKeyNode.publicKeyBase58 = this.publicKeyBase58;
     return publicKeyNode;
   }
 
+  /**
+   * Adds an encrypted private key to the KeyPair.
+   * Used by LDKeyPair.export(), to serialize public + private key pair.
+   *
+   * Usage:
+   *
+   * ```
+   * await keyPair.export();
+   * // ->
+   * {
+   *   id: 'did:example:1234#z6LSjeJZaUHMvEKW7tEJXV4PrSm61NzxxHhDXF6zHnVtDu9g',
+   *   type: 'X25519KeyAgreementKey2019',
+   *   controller: 'did:example:1234',
+   *   publicKeyBase58: '8y8Q4AUVpmbm2VrXzqYSXrYcAETrFgX4eGPJoKrMWXNv',
+   *   privateKeyBase58: '95tmYuhqSuJqY77FEg78Zy3LFQ1cENxGv2wMvayk7Lqf'
+   * }
+   * ```
+   *
+   * @param {object} keyNode - A plain object.
+   *
+   * @return {object} The keyNode with an encrypted private key attached.
+   */
+  async addEncryptedPrivateKey(keyNode) {
```

Comment:
This is unrelated to this PR (I think) ... don't know what it's about, ignoring.

Reply:
Yeah (it's the method that enables `.export()` to work).